.github/workflows/copy-cloud-docs-for-tfe.yml

```diff
@@ -0,0 +1,219 @@
+          echo ""                                                                                     >> $GITHUB_STEP_SUMMARY
+          echo "**Series**: ${{env.SERIES}}"                                                          >> $GITHUB_STEP_SUMMARY
+          echo "**Release**: ${{env.RELEASE}}"                                                        >> $GITHUB_STEP_SUMMARY
+          echo ""                                                                                     >> $GITHUB_STEP_SUMMARY
+          echo "---"                                                                                  >> $GITHUB_STEP_SUMMARY
+
+      - name: Checkout new release branch
+        uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4.1.4
+        with:
+          ref: ${{env.release_branch}}
+          path: "${{github.workspace}}/release"
+
+      - name: Checkout main for new docs version ACTUAL PR
+        uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4.1.4
+        with:
+          path: "${{github.workspace}}/new-docs-pr"
+
+      - name: Checkout main for new docs version DIFF PR
+        uses: actions/checkout@0ad4b8fadaa221de15dcec353f45205ec38ea70b # v4.1.4
+        with:
+          path: "${{github.workspace}}/new-docs-diff-pr"
+
+      - name: Generate version-metadata for workflow
+        working-directory: "${{github.workspace}}/workflow"
+        run: |
+          npm i
+          npm run prebuild-only-version-metadata
+
+      # MAKING THE DIFF PR
+      - name: Copy files for new docs version DIFF PR
+        uses: ./workflow/.github/actions/copy-cloud-docs-for-tfe
+        with:
+          source_path: "${{github.workspace}}/release"
+          target_path: "${{github.workspace}}/new-docs-diff-pr"
+
+      - name: Open new docs docs version DIFF PR
+        working-directory: "${{github.workspace}}/new-docs-diff-pr"
+        env:
+          branch_name: docs-diff/${{env.SERIES}}-${{env.RELEASE}}
+          pr_body: |
+            # Automated Docs DIFF PR for TFE ${{env.SERIES}}-${{env.RELEASE}}
+
+            **❗ DO NOT MERGE THIS PR, IT IS FOR DIFFS ONLY ❗**
+
+            **TFE Series**: ${{ env.SERIES }}
+            **TFE Release**: ${{ env.RELEASE }}
+
+            This copies over `cloud-docs` from:
+            - ${{github.server_url}}/${{github.repository}}/tree/${{env.release_branch}}
+
+            This PR was created via:
+            - ${{github.server_url}}/${{github.repository}}/actions/runs/${{github.run_id}}
+
+            Triggered by creation of branch:
+            - ${{github.server_url}}/${{github.repository}}/tree/${{env.release_branch}}
+
+        # secrets.WORKFLOW_TESTING_TOKEN requires permissions read:org, repo, workflow
+        run: |
+          echo ${{ secrets.WORKFLOW_TESTING_TOKEN }} | gh auth login --with-token
+          git config --global user.email "team-rel-eng@hashicorp.com"
+          git config --global user.name "tfe-release-bot"
```

Comment:
Do we want these to be env vars?

Reply:
We don't use them as env vars in other workflow files: https://github.com/hashicorp/web-unified-docs/blob/main/scripts/tfe-releases/ci/execute-changelog-script.sh#L48